api/dashboard/list_user.go

```diff
@@ -0,0 +1,102 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package dashboard
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"github.com/sirupsen/logrus"
+	"gorm.io/gorm"
+
+	"github.com/go-vela/server/api/types"
+	"github.com/go-vela/server/database"
+	"github.com/go-vela/server/router/middleware/user"
+	"github.com/go-vela/server/util"
+)
+
+// swagger:operation GET /api/v1/user/dashboards dashboards ListUserDashboards
+//
+// Get all dashboards for the claims users in the configured backend
```

Comment:
nit: 
```suggestion
// Get all dashboards for the current user in the configured backend
```

Reply:
i know we use "claims user" elsewhere so i would propose updating there too. this reads better, imo.